Kudu.Services.Web/Content/Scripts/ProcessExplorer.js

```diff
@@ -516,7 +516,7 @@ var Process = (function () {
         }));
 
         buttonDiv.appendChild(Utilities.getButton("ui-button-info", div.id + "-dumb", "Download memory dump", function () {
```

Comment:
Should "-dumb" be "-dump" ?

Reply:
I checked if that identifier is used anywhere else and since it's not I simply changed it.